Cpix/HlsPlaylistType.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Axinom.Cpix
+{
+	public class HlsPlaylistType
+	{
+		public const string Master = "master";
+		public const string Variant = "variant";
```

Comment:
This will be changed to Master and Media in response to the bug you filed. You can already use the new names here. Maybe you can even update the XSD here and make a PR in Cpix spec repo to make the correction to the official XSD so it matches the library?

Reply:
I'll update here and also look at making a PR for Cpix spec.